src/main/java/de/uhd/ifi/se/decision/management/jira/rest/impl/ConfigRestImpl.java

```diff
@@ -196,6 +197,31 @@ public Response getKnowledgeTypes(@QueryParam("projectKey") String projectKey) {
 	return Response.ok(knowledgeTypes).build();
     }
 
+    @Override
+    @Path("/getDecisionGroups")
+    @GET
+    public Response getDecisionGroups(@QueryParam("elementId") long id, @QueryParam("location") String location,
+	    @QueryParam("projectKey") String projectKey) {
+	KnowledgeElement element = KnowledgePersistenceManager.getOrCreate(projectKey).getDecisionKnowledgeElement(id,
+		location);
+	if (element != null) {
+	    List<String> groups = element.getDecisionGroups();
+	    if (groups != null) {
+		for (String group : groups) {
+		    if (group.equals("High_Level") || group.equals("Medium_Level")
+			    || group.equals("Realization_Level")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5160185)